02-express-tutorial/app.js

```diff
@@ -1,15 +1,40 @@
 const express = require("express");
 const path = require("path");
-const { products } = require("./data");
-console.log("Express Tutorial");
+const { products, people } = require("./data");  
+const peopleRouter = require("./routes/people");
```

Comment:
The `peopleRouter` is imported here but not used

Reply:
I utilized the imported peopleRouter by adding it to app.js with app.use("/api/v1/people", peopleRouter) to ensure it's used for handling the relevant routes.